### PR TITLE
fix(evm): Added Cancun and Shanghai blocks to ChainConfig

### PR DESCRIPTION
**Pull request by Vvaradinov**
Adds the Shanghai and Cancun blocks to the `ChainConfig`

Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: proto/ethermint/evm/v1/evm.proto [hunk 1]
```diff
@@ -117,6 +117,17 @@ message ChainConfig {
...
+  string shanghai_block = 22 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"shanghai_block\""
+  ];
+  // cancun_block switch block (nil = no fork, 0 = already on cancun)
+  string cancun_block = 23 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"cancun_block\""
+    ];
+
```
Comment (fedekunze) [inline proto/ethermint/evm/v1/evm.proto:130]: Can you fix the padding?

**Review by fedekunze — APPROVED** [review]
LGTM, minor comment

**Comment by fedekunze** [comment]
@Vvaradinov can you add a changelog entry?

**Review by danburck — APPROVED** [review]
@Vvaradinov LGTM! Having these smaller PRs is great

**Pull request merged**